src/fiatExchanges/quotes/FiatConnectQuote.ts

```diff
@@ -1,3 +1,4 @@
+import { FiatConnectApiClient } from '@fiatconnect/fiatconnect-sdk'
```

Comment:
Remove `fiatConnectClient` as an attribute from the class and get rid of this import?

Reply:
good catch!